components/Common/Search/States/WithAllResults.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Results } from '@orama/orama';
+import NextLink from 'next/link';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import styles from './index.module.css';
+
+import type { SearchDoc } from '@/components/Common/Search/States/SearchBox';
```

Comment:
I have the feeling the type import should happen before the styles one.

Reply:
I am following the linter... let me try